unlock-protocol-com/src/components/layout/DefaultLayout.tsx

```diff
@@ -287,8 +287,10 @@ export function Layout({ children }: Props) {
       </div>
 
       {children}
-      <div className={`${containerClass} md:pt-14 pt-4`}>
-        <Footer {...FOOTER} />
+      <div className="px-6">
```

Comment:
Are we applying same padding on bigger screen in other components too? Just to confirm.

Reply:
yes!